source/crud/tests/v2/updateWithPipelines.yml

```diff
@@ -40,6 +48,14 @@ tests:
         arguments:
             filter: {}
             update: [ { $project: { x: 1 } }, { $addFields: { foo: 1 } } ]
+        result:
+          deletedCount: 0
+          insertedCount: 0
+          insertedIds: {}
+          matchedCount: 2
+          modifiedCount: 2
+          upsertedCount: 0
+          upsertedIds: {}
```

Comment:
Same suggestions as above for `updateOne`.

Reply:
Done